.ci/cico_rhche_prcheck.sh

```diff
@@ -136,6 +132,8 @@ else
 fi
 set -x
 
+oc policy add-role-to-user edit kkanova rhopp tdancs garagatyi ibuziuk amisevsk dfestal -n prcheck-${RH_PULL_REQUEST_ID}
```

Comment:
My GitHub user name (which is used in dev-cluster now afaict) is `davidfestal` (instead of `dfestal`)

Reply:
Copy that!

Reply:
@davidfestal Fixed.